thor/config.go

```diff
@@ -30,17 +31,18 @@ type Config struct {
 	ValidatorEvictionThreshold uint32 `json:"validatorEvictionThreshold"` // the number of blocks after which offline validator will be evicted from the leader group (7 days)
 
 	// staker parameters
-	LowStakingPeriod    uint32 `json:"lowStakingPeriod"`
-	MediumStakingPeriod uint32 `json:"mediumStakingPeriod"`
-	HighStakingPeriod   uint32 `json:"highStakingPeriod"`
-	CooldownPeriod      uint32 `json:"cooldownPeriod"`
+	LowStakingPeriod    uint32  `json:"lowStakingPeriod"`
+	MediumStakingPeriod uint32  `json:"mediumStakingPeriod"`
+	HighStakingPeriod   uint32  `json:"highStakingPeriod"`
+	CooldownPeriod      uint32  `json:"cooldownPeriod"`
+	HayabusaTP          *uint32 `json:"hayabusaTP"`
 }
 
 // SetConfig sets the config.
 // If the config is not set, the default values will be used.
 // If the config is locked, will panic.
 func SetConfig(cfg Config) {
-	if locked {
+	if locked && cfg.HayabusaTP == nil {
```

Comment:
```suggestion
	if locked {
```

If there is a panic in tests, we can add a new customnet function that does not call `LockConfig`